Resources/Styles/ControlStyles.xaml

```diff
@@ -629,4 +629,133 @@
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewCanvasRectangleStyle" TargetType="{x:Type Rectangle}">
+        <Setter Property="Width" Value="140" />
+        <Setter Property="Height" Value="35" />
+        <Setter Property="Fill" Value="White" />
+        <Setter Property="Stroke" Value="Black" />
+        <Setter Property="StrokeThickness" Value="2" />
+        <Setter Property="Canvas.Left" Value="260" />
+        <Setter Property="Canvas.Top" Value="76" />
+        <Setter Property="HorizontalAlignment" Value="Left" />
+        <Setter Property="VerticalAlignment" Value="Top" />
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewPayeeStyle" TargetType="{x:Type TextBlock}">
+        <Setter Property="Grid.Row" Value="1" />
+        <Setter Property="Grid.Column" Value="0" />
+        <Setter Property="Grid.ColumnSpan" Value="1" />
+        <Setter Property="FontStyle" Value="Italic" />
+        <Setter Property="FontSize" Value="9" />
+        <Setter Property="IsEnabled" Value="False" />
+        <Setter Property="Margin" Value="22,10,173,1" />
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewWordCurrencyStyle" TargetType="{x:Type TextBlock}">
+        <Setter Property="Grid.Row" Value="2" />
+        <Setter Property="Grid.Column" Value="0" />
+        <Setter Property="Grid.ColumnSpan" Value="1" />
+        <Setter Property="FontStyle" Value="Italic" />
+        <Setter Property="FontSize" Value="9" />
+        <Setter Property="TextWrapping" Value="Wrap" />
+        <Setter Property="IsEnabled" Value="False" />
+        <Setter Property="Margin" Value="22,0,173,6" />
+        <Setter Property="Width" Value="45" />
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewDateStyle" TargetType="{x:Type TextBlock}">
+        <Setter Property="Grid.Row" Value="0" />
+        <Setter Property="Grid.Column" Value="1" />
+        <Setter Property="Grid.ColumnSpan" Value="1" />
+        <Setter Property="FontWeight" Value="Normal" />
+        <Setter Property="FontSize" Value="9" />
+        <Setter Property="IsEnabled" Value="False" />
+        <Setter Property="Margin" Value="20,29,0,0" />
+        <Setter Property="Height" Value="15" />
+        <Setter Property="Width" Value="20" />
+        <Setter Property="HorizontalAlignment" Value="Left" />
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewNumericCurrencyStyle" TargetType="{x:Type TextBlock}">
+        <Setter Property="Grid.Row" Value="2" />
+        <Setter Property="Grid.Column" Value="1" />
+        <Setter Property="Grid.ColumnSpan" Value="1" />
+        <Setter Property="FontWeight" Value="Bold" />
+        <Setter Property="IsEnabled" Value="False" />
+        <Setter Property="Margin" Value="30,14,0,0" />
+        <Setter Property="Width" Value="20" />
+        <Setter Property="HorizontalAlignment" Value="Left" />
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewWordBelowSignLineStyle" TargetType="{x:Type TextBlock}">
+        <Setter Property="Grid.Row" Value="3" />
```

Comment:
The `Grid.Row`, `Grid.Column` and `Grid.ColumnSpan` is only available to use when there is a Grid as parent control. This violates the code reuse if this TextBlock is not in a Grid layout control.

Try to define the row, column and the related "span" on View is suitable already.